docs/architecture/PLATFORM_TRACK_DEV_PLAN.md

```
@@ -0,0 +1,221 @@
+| 根目录 `.md` | 10 | 10 | ≤10 ✅ |
+| 真孤儿 | 0 | 0 | 0 ✅ |
+| 净行数 vs main | — | **+490 / -2102 = -1612** | 各 Phase PR ≤0 |
+| net-change CI | warning >200 | 本批净减 ✅ | fail if NET > 0 |
+
+**阻塞 PL-0 合 main**：services 仍 **251**，需再删或内联 **≥1** 个 `.py`；未 commit；CI 未在 PR 上复验。
+
+---
+
+## 条目分解与执行顺序
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)